docs/getting_started/evaluate.md

```diff
@@ -0,0 +1,85 @@
+After the model has been trained, it can be used to predict the target output features on new data.
+
+We've created a small test dataset containing input features for 10 movie reviews that we can use for testing. Click this [link](https://github.com/ludwig-ai/ludwig-docs/raw/getting-started/docs/data/rotten_tomatoes_test.csv) to download the test dataset.
+
+Now, let's make some predictions on the test dataset (Note: if you are working in a notebook, you can use the model created in the training section):
```

Comment:
Essentially I was saying that they don't need to call the model.load(<path/to/model>) line because the model was defined in the previous step, though this applies to all python not just notebooks, so I'm thinking about just removing line 16 altogether. WDYT?

Reply:
Makes sense.

I've suggested a change to include this detail as a comment in Python.

If that works for you, then we can change this line to just say: "Let's make some predictions on the test dataset!"